index.html

```diff
@@ -103,14 +102,20 @@ <h1 class="categories">Welcome to my portfolio!</h1>
       by Marshall Miller
     </footer>
 
+    <script src="vendors/js/page.js"></script>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/handlebars.js/4.0.5/handlebars.js"></script>
+    <script src="js/aboutController.js"></script>
+    <script src="js/schoolController.js"></script>
+    <script src="js/workController.js"></script>
+    <script src="js/projectController.js"></script>
+    <script src="js/routes.js"></script>
     <script src="js/tabSlide.js"></script>
     <script src="js/portfolio.js"></script>
     <script>
```

Comment:
You don't need to keep the <script> tags in the index when you have routing. You could easily move these function calls to your controller where they belong. Just be sure to consider what order your JS files should be in.